cloudify_cli/commands/instances.py

```diff
@@ -0,0 +1,78 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+"""
+Handles all commands that start with 'cfy instances'
+"""
+
+from cloudify_rest_client.exceptions import CloudifyClientError
+from cloudify_cli import utils
+from cloudify_cli.exceptions import CloudifyCliError
+from cloudify_cli.logger import get_logger
+
+
+def get(node_instance_id):
```

Comment:
overall, the two "get" and two "ls" methods you implemented are similar and are also probably similar to those found in "executions" and "workflows".
it might have been a good idea to consider refactoring these to some common method, to help in maintenance for this code.